src/features/game/types/achievements.ts

```diff
@@ -120,6 +132,8 @@ export const ACHIEVEMENTS: () => Record<AchievementName, Achievement> = () => ({
       return bakedCakes;
     },
     requirement: 10000,
```

Comment:
is this really the requirement for "Bake 13 cakes" ? or is it should be 13?

Reply:
Eagle eye 👁️ 🦅 !!!

You are right, we will need to fix this